packages/botonic-react/src/webchat/header/styles.ts

```diff
@@ -9,9 +9,13 @@ export const HeaderContainer = styled.div`
     ${COLORS.BLEACHED_CEDAR_PURPLE} 0%,
     ${props => props.color} 100%
   );
-  border-radius: ${WEBCHAT.DEFAULTS.BORDER_RADIUS_TOP_CORNERS};
+  border-radius: ${props => props.theme.header?.style?.borderRadius};
   z-index: 2;
   height: inherit;
+
+  @media (max-width: ${props => props.theme.mobileBreakpoint}px) {
+    border-radius: 0px;
```

Comment:
Doubt: Shouldn't this value be taken from the `defaultTheme` -> `mobileStyle` properties?

Reply:
Yes, I can add a header.mobileStyle.borderRadius with a value in defaultTheme and then use it from `theme.header?.mobileStyle?.borderRadius`

Reply:
applied